homeassistant/components/matter/strings.json

```diff
@@ -88,6 +88,12 @@
       },
       "boost_state": {
         "name": "Boost state"
+      },
+      "dishwasher_alarm_inflow": {
```

Comment:
Side note: In a separate PR I'd suggest we order the translation strings 🔡 per platform. Now they are ordered per device more or less but that doesn't help when you try to find them in this file since you don't have the device names available here.